core/common/src/grpc_interceptor.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use bytes::Bytes;
+use core::future::Future;
+use futures_core::task::{Context, Poll};
+use http::uri::Uri;
+use http_body::Body;
+use std::error::Error;
+use std::pin::Pin;
+use tower::{Layer, Service};
+
+/// This module provides the gRPC Interceptor construct. It can be used to
```

Comment:
`///` is for a doc comment. Do you know where this documentation is being generated? I'm concerned that this will be added to the docs for the `GRPC_HEADER_LENGTH` constant which is probably not want you want. Please confirm what these comments are documenting, and consider using either normal comments (`//`) or migrating this to a README

Reply:
I will use //